browser/app/js/buckets/__tests__/actions.test.js

```diff
@@ -40,32 +48,54 @@ describe("Buckets actions", () => {
     })
   })
 
-  it("creates buckets/SET_LIST directly", () => {
+  it("should update browser url and creates buckets/SET_CURRENT_BUCKET action when selectBucket is called", () => {
     const store = mockStore()
     const expectedActions = [
-      { type: "buckets/SET_LIST", buckets: ["test1", "test2"] }
+      { type: "buckets/SET_CURRENT_BUCKET", bucket: "test1" }
     ]
-    store.dispatch(actionsBuckets.setList(["test1", "test2"]))
+    store.dispatch(actionsBuckets.selectBucket("test1"))
     const actions = store.getActions()
     expect(actions).toEqual(expectedActions)
+    expect(window.location.pathname).toBe("/test1")
   })
 
-  it("creates buckets/SET_FILTER directly", () => {
+  it("creates buckets/SHOW_MAKE_BUCKET_MODAL for showMakeBucketModal", () => {
     const store = mockStore()
-    const expectedActions = [{ type: "buckets/SET_FILTER", filter: "test" }]
-    store.dispatch(actionsBuckets.setFilter("test"))
+    const expectedActions = [
+      { type: "buckets/SHOW_MAKE_BUCKET_MODAL", show: true }
+    ]
+    store.dispatch(actionsBuckets.showMakeBucketModal())
     const actions = store.getActions()
     expect(actions).toEqual(expectedActions)
   })
 
-  it("should update browser url and creates buckets/SET_CURRENT_BUCKET action when selectBucket is called", () => {
+  it("creates buckets/SHOW_MAKE_BUCKET_MODAL for hideMakeBucketModal", () => {
     const store = mockStore()
     const expectedActions = [
-      { type: "buckets/SET_CURRENT_BUCKET", bucket: "test1" }
+      { type: "buckets/SHOW_MAKE_BUCKET_MODAL", show: false }
     ]
-    store.dispatch(actionsBuckets.selectBucket("test1"))
+    store.dispatch(actionsBuckets.hideMakeBucketModal())
     const actions = store.getActions()
     expect(actions).toEqual(expectedActions)
-    expect(window.location.pathname).toBe("/test1")
+  })
+
+  it("creates buckets/ADD action", () => {
+    const store = mockStore()
+    const expectedActions = [{ type: "buckets/ADD", bucket: "test" }]
+    store.dispatch(actionsBuckets.addBucket("test"))
+    const actions = store.getActions()
+    expect(actions).toEqual(expectedActions)
+  })
+
+  it("creates buckets/SHOW_MAKE_BUCKET_MODAL buckets/ADD and buckets/SET_CURRENT_BUCKET after creating the bucket", () => {
```

Comment:
As far as I can see we aren't creating buckets/SHOW_MAKE_BUCKET_MODAL here. If that's the case, it might be better to remove this from the string to avoid confusion

Reply:
yes, will remove.